### PR TITLE
Add missing badges in user profile for /projects and /packages

### PR DESCRIPTION
**Pull request by jakob30061**
- Implement fix for user and organization page
- Add necessary ctx.Data

 When switching tabs on `user profile page` the badges are shown for `/gitea_user?tab=stars`, `/gitea_user?tab=activity` and missing under `gitea_user/-/projects` and `gitea_user/-/packages`

This was added for users and organizations.

Before:
image
After:
image

### CODE REVIEW

File: routers/web/org/members.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ func Members(ctx *context.Context) {
 	}
 	ctx.Data["Page"] = pager
 	ctx.Data["Members"] = members
+	ctx.Data["Owner"] = ctx.ContextUser
```
Comment (wxiaoguang) [inline routers/web/org/members.go]: After grepping the code , I think the "Owner" concept here is wrong (not by this PR, but it's a legacy problem). It should be "ContextUser", and the templates should also use the "ContextUser" variable.

Comment (wxiaoguang) [reply routers/web/org/members.go]: I think we need this first:  Fix unclear "Owner" concept #24233

Comment (jakob30061) [reply routers/web/org/members.go]: I will look into it. 
There are more places ctx.ContextUser is used outside of this PR. 

These should be updated as well.

Comment (wxiaoguang) [reply routers/web/org/members.go]: Only a few legacy "Owner" variables, I think they could be fixed by  #24233

Comment (jakob30061) [reply routers/web/org/members.go]: @wxiaoguang  Do I have to update my PR as well?

Comment (wxiaoguang) [reply routers/web/org/members.go]: I think wen can wait for #24233 first, then maybe some `ctx.Data` code is not needed anymore, the `ContextUser` could already have been prepared by the middleware handlers already, just like the deleted `ctx.Data["Owner"]=org` in #24233 .

Comment (silverwind) [reply routers/web/org/members.go]: #24233 was merged.

Comment (jakob30061) [reply routers/web/org/members.go]: I rebased the PR and updated "Owner" to "ContextUser". 
Two ctx.Data could be removed. 
ContextUser was already prepared by the middleware handlers.

**Review by yardenshoham — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Comment by silverwind** [comment]
.

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Comment by techknowlogick** [comment]
🤖 🎺

**Pull request merged**